.github/workflows/test-build-docker-image.yml

```
@@ -1336,7 +1336,7 @@ jobs:
           docker buildx build \
             --platform linux/arm64,linux/amd64 \
             --push \
-            --build-arg APPSMITH_SEGMENT_CE_KEY=${{ secrets.APPSMITH_SEGMENT_CE_KEY_RELEASE }} \
+            --build-arg GIT_SHA=${GITHUB_SHA} APPSMITH_SEGMENT_CE_KEY=${{ secrets.APPSMITH_SEGMENT_CE_KEY_RELEASE }} \
```

This needs another `--build-arg`, no?

```suggestion
            --build-arg GIT_SHA="${GITHUB_SHA}" \
            --build-arg APPSMITH_SEGMENT_CE_KEY="${{ secrets.APPSMITH_SEGMENT_CE_KEY_RELEASE }}" \
```